### DISCUSSION TITLE
Cannot disable labels for array and blocks

### DISCUSSION THREAD

**Discussion by iakshay** [payloadcms/payload#1742] [General]
It seems like currently there is no way to disable label for array and blocks. I have tried using `label: false` on latest payload version for `blocks` and `array` and I still see `h3`.

`label: false` does work for Group though. Would be nice to have consistent label behavior.

Maybe we can update this line (for blocks), to only show label if one is set ([similar to Group](https://github.com/payloadcms/payload/blob/master/src/admin/components/forms/field-types/Group/index.tsx#L64-L66))
https://github.com/payloadcms/payload/blob/master/src/admin/components/forms/field-types/Blocks/index.tsx#L227

**Comment by jacobsfletch** [comment]
@iakshay totally, I think you're spot on. For now, you might be able to set the label as an empty space, like this:

```ts
{
  type: 'blocks',
  label: ' '
}
```

But looking at the source code, you'll still get the `h3`. This should be pretty easy to disable.

**Reply by DanRibbens** [reply]
To add to to this, you could add custom CSS that hides the h3 also. You'll want to be selective with the path id if you do this unless you want to remove all of them.

I agree, we should support disabling labels on all fields.
